core-java/src/main/java/com/baeldung/nulls/PrimitivesAndWrapper.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.nulls;
+
+public class PrimitivesAndWrapper {
+
+    public static int sum(int a, int b) {
+        return a + b;
+    }
+
+    public static Integer sum(Integer a, Integer b) {
+        return a + b;
+    }
+
+    public static Integer goodSum(Integer a, Integer b) {
+        if (a != null && b != null)
+            return a + b;
+        else
+            throw new IllegalArgumentException();
+    }
+
+    public static void main(String[] args) {
```

Comment:
Write a unit test instead, and test each method. For the ones using wrappers, write a test without nulls and assert the result, and then write a test with one or more nulls and assert an expected exception.